### PR TITLE
suggestions for the software corner article

### PR DESCRIPTION
**Pull request by garthtarr**

### CODE REVIEW

**Comment by garthtarr** [comment]
why so many tau values, it appears that it's only the largest tau level that is used in the article, maybe stick with that and not report the others?

**Comment by garthtarr** [comment]
Should Samuel be an author too? Have you discussed this with him?

**Comment by garthtarr** [comment]
The plots use x1 x2 etc, but the table output uses variable names. I assume this is because you can't fit the variable names inside the circle, but might be more user friendly if you include the variable names in the linkage plots, even if it means the text needs to be below the circles

**Comment by garthtarr** [comment]
Other comments in square brackets in line with text.

File: software_corner.Rmd [hunk 1]
```diff
@@ -67,26 +64,35 @@ One way to "diagnose" multicollinearity is to compute the eigenvalues of the mat
...
 
 ```{r}
+# install.packages("mcvis")
 library(mcvis)
 mcvis_result = mcvis(X)
 print(mcvis_result)
 ```
 
-In the output above, we see that $X_1$, the first column of $X$, is identified as the main variable causing multicollinearity. We can further visualise this via a (bipartite) graph that shows this connection between $\tau_p$ and the $p$ predictor variables. The size and colour of the lines are categorised by the magnitude of the MC-index value. This plot gives us a quick indication of the sources of the multicollinearity, ordered by the magnitude. 
+In the output above, we see that $X_1$, the first column of $X$, is identified as the main variable causing multicollinearity. [Why? what is it about the table that indicates this? there are larger values in the X05 column for example. Or should we only look in the tau5 row?]
```
Comment (kevinwang09) [inline software_corner.Rmd:92]: The `print.mcvis` method implemented in the devel version of mcvis, which only prints tau_p

File: software_corner.Rmd [hunk 2]
```diff
@@ -67,26 +64,35 @@ One way to "diagnose" multicollinearity is to compute the eigenvalues of the mat
...
 library(mcvis)
 mcvis_result = mcvis(X)
 print(mcvis_result)
 ```
 
-In the output above, we see that $X_1$, the first column of $X$, is identified as the main variable causing multicollinearity. We can further visualise this via a (bipartite) graph that shows this connection between $\tau_p$ and the $p$ predictor variables. The size and colour of the lines are categorised by the magnitude of the MC-index value. This plot gives us a quick indication of the sources of the multicollinearity, ordered by the magnitude. 
+In the output above, we see that $X_1$, the first column of $X$, is identified as the main variable causing multicollinearity. [Why? what is it about the table that indicates this? there are larger values in the X05 column for example. Or should we only look in the tau5 row?] 
+We can further visualise this via a (bipartite) graph that shows this connection between $\tau_p$ and the $p$ predictor variables. 
+The size and colour of the lines are categorised by the magnitude of the MC-index value. 
+This plot gives us a quick indication of the sources of the multicollinearity, ordered by the magnitude. [The thickness of the line represents magnitude?]
```
Comment (kevinwang09) [inline software_corner.Rmd:95]: The `plot.mcvis` method implemented in the devel version of mcvis is a different plot.

**Pull request merged**